### DISCUSSION TITLE
PaddleOCR什么时候支持SLANet_plus的微调

### DISCUSSION THREAD

**Discussion by yazheng0307** [PaddlePaddle/PaddleOCR#13944] [Q&A] [answered]
目前只能在PaddleX中使用

**Comment by kerry-weic** [comment]
+1,看PaddleX中有相关的微调[文档](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/table_structure_recognition.md)也提供了[SLANet_plus](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/paddlex/configs/table_recognition/SLANet_plus.yaml)的配置文件，看配置文件和 paddleocr预训练的文件格式还不太一样，还不清楚是否能共用

**Comment by archwolf118** [comment]
slanet_plus的改进在哪些方面，能否赐教？谢谢！

**Reply by GreatV** [reply]
具体实现请看paddlex仓库

**Comment by GreatV** [comment]
目前需要使用paddlex，paddleocr没有计划。

**Reply by hustzxl** [reply]
paddlex里面也不能微调吧 训练模型没放出来，放出来的是推理模型

**Reply by GreatV** [reply]
请参考 https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/table_structure_recognition.md

**Reply by cuicheng01** [reply, marked as answer]
您好，训练模型是放出来的，这块paddlex的文档也在优化中，近期文档会更新模型下载链接。您需要的模型是[这个](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/paddlex/configs/table_recognition/SLANet_plus.yaml#L32)，可以下载使用。另外，这块PaddleX也在研发更强的表格识别模型，敬请关注。
